### DISCUSSION TITLE
How to install Windows 11 without TPM from boot media?

### DISCUSSION THREAD

**Discussion by Nicos18** [AveYo/MediaCreationTool.bat#150] [Q&A] [answered]
Hi,

In the old builds of MediaCreationTool.bat, there was a subfolder called "MCT" and then if I wanted to install Windows 11 booting from the media, I'd selected the "Skip_TPM_Check_on_Media_Boot.

Now this isn't available in the new builds of the project.

I'd like to know if that .bat is compatible with the 22H2 builds or Windows 11 and, if not, how can I install Windows 11 from boot media skipping TPM checks?

Thanks

**Comment by AveYo** [comment, marked as answer]
Everything is in the readme. The media created by the script bypasses requirements on boot as well, you don't need to do anything.

For Dev builds that you've downloaded elsewhere, you could use the [Quick_11_iso_esd_wim_TPM_toggle.bat](https://github.com/AveYo/MediaCreationTool.bat/blob/main/bypass11/Quick_11_iso_esd_wim_TPM_toggle.bat) - directly on the `.iso` file, or on `sources\install.wim` if it's already on USB stick / drive.
_Launch the script once, then can right-click -- `Send to` on any iso / wim / esd and toggle bypass on._

Alternatively can add the [AutoUnattend.xml](https://github.com/AveYo/MediaCreationTool.bat/blob/main/bypass11/AutoUnattend.xml) to the USB stick / drive you've made, it will also bypass requirements _(via reg keys, less reliable)_.
